app/dashboard/template.hbs

```diff
@@ -14,28 +14,6 @@
         create=(perform createNode)
     }}
 {{/if}}
-<div class="prereg-banner">
```

Comment:
Probably should move this to a component which isn't used yet instead of deleting it outright

Reply:
I can do this, but I'm not sure I understand why, which would affect how I go about making the component. The prereg challenge is winding down, and we're removing the banner because we aren't soliciting for new challengers anymore. Is it to be prepared for potential new banners?